src/plotman/_tests/archive_test.py

```diff
@@ -8,6 +8,7 @@ def test_compute_priority():
 def test_rsync_dest():
     arch_dir = '/plotdir/012'
     arch_cfg = configuration.Archive(
+        rsyncd_type='drives'
```

Comment:
Might appease tests:
```suggestion
        rsyncd_type='drives',
```